woo-razorpay.php

```diff
@@ -1684,10 +1688,12 @@ function check_razorpay_response()
 
             // If the order has already been paid for
             // redirect user to success page
-            if ($order->needs_payment() === false)
+            if (($order->needs_payment() === false) and 
+                ($orderStat != 'draft') and 
+                ($orderStat != 'wc-checkout-draft'))
```

Comment:
@poojarazor can we use in_array() here

Reply:
@abdulwahidsharief  don't want to initialise the array for the statement